src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsJsonEncoding.cs

```diff
@@ -306,7 +306,7 @@ LaunchProfile ReadProfile(string name)
                 {
                     JsonToken.Boolean => (bool)reader.Value!,
                     JsonToken.Integer => checked((int)(long)reader.Value!),
-                    JsonToken.StartObject => (jsonSerializer ??= JsonSerializer.CreateDefault()).Deserialize<Dictionary<string, string>>(reader),
+                    JsonToken.StartObject => (jsonSerializer ??= JsonSerializer.CreateDefault()).Deserialize<Dictionary<string, object>>(reader),
```

Comment:
>object [](http://example.com/codeflow?start=128&length=6)

Are there any consumers of this that were mistakenly expecting string?

Reply:
I couldn't find any. It's a loose contract though so it's hard to know for sure. I expect not, as this regressed in 17.3, meaning earlier versions would have received object.